specification/storage/data-plane/Microsoft.FileStorage/stable/2025-11-05/file.json

```diff
@@ -0,0 +1,8681 @@
+              "ETag": {
+                "type": "string",
+                "description": "The ETag contains a value which represents the version of the file, in quotes."
+              },
+              "Last-Modified": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "Returns the date and time the directory was last modified. Any operation that modifies the share or its properties or metadata updates the last modified time. Operations on files do not affect the last modified time of the share."
+              },
+              "x-ms-content-crc64": {
+                "type": "string",
+                "format": "byte",
+                "description": "This header is returned so that the client can check for message content integrity. The value of this header is computed by the File service; it is not necessarily the same value as may have been specified in the request headers."
+              },
+              "x-ms-request-id": {
+                "x-ms-client-name": "RequestId",
+                "type": "string",
+                "description": "This header uniquely identifies the request that was made and can be used for troubleshooting the request."
+              },
+              "x-ms-version": {
+                "x-ms-client-name": "Version",
+                "type": "string",
+                "description": "Indicates the version of the File service used to execute the request."
+              },
+              "Date": {
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated."
+              },
+              "x-ms-request-server-encrypted": {
+                "x-ms-client-name": "IsServerEncrypted",
+                "type": "boolean",
+                "description": "The value of this header is set to true if the contents of the request are successfully encrypted using the specified algorithm, and false otherwise."
+              },
+              "x-ms-file-last-write-time": {
+                "x-ms-client-name": "FileLastWriteTime",
+                "type": "string",
+                "format": "date-time-rfc1123",
+                "description": "Last write time for the file."
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              },
+              "x-ms-copy-source-error-code": {
+                "x-ms-client-name": "CopySourceErrorCode",
+                "type": "string"
+              },
+              "x-ms-copy-source-status-code": {
+                "x-ms-client-name": "CopySourceStatusCode",
+                "type": "integer",
+                "format": "int64"
```

Comment:
If this is for HTTP status codes, don't make it `int64`.  [They can have 3 digits max.](https://httpwg.org/specs/rfc9110.html#rfc.section.15)

Reply:
Ditto for other use cases here and in blobs.

Reply:
Fixed.